### PR TITLE
fix rerunning servicemesh test

### PR DESCRIPTION
**Pull request by maschmid**

### CODE REVIEW

File: test/servinge2e/servicemesh_test.go [hunk 1]
```diff
@@ -336,6 +337,16 @@ func jwtHttpGetRequestBytes(url string, token *string) (*http.Response, []byte,
 	return resp, body, err
 }
 
+// randomSuffix generates a random suffix suitable for making unique k8s resource names
+func randomSuffix(t *testing.T) string {
```
Comment (mgencur) [inline test/servinge2e/servicemesh_test.go:341]: We could probably reuse https://github.com/knative/pkg/blob/master/test/helpers/name.go#L59

Comment (maschmid) [reply test/servinge2e/servicemesh_test.go:341]: OK, I'll wait for https://github.com/openshift-knative/serverless-operator/pull/432 first then, as that would conflict with the Gopkg and vendoring changes there.

**Comment by rhuss** [comment]
From my naive perspective that looks good (tests are passing, too), so I'm fine if @markusthoemmes has no objections.

**Comment by matzew** [comment]
@maschmid do you need this also on the (fresh) `release-1.9` branch ? 

Currently the PR for "CI runs" on that branch has _not_ been merged, yet /cc @mgencur

**Comment by maschmid** [comment]
@matzew it is not critical to backport this for CI, these tests are skipped by default, as we don't install servicemesh on CI (and the problem this PR fixes manifests only when rerunning the test on the same cluster, so it won't affect CI either way... )

I can open a new backport PR for release-1.9 once this one lands in master.

**Pull request merged**